packages/build-utils/package.json

```diff
@@ -13,8 +13,8 @@
   "scripts": {
     "build": "node build",
     "test": "jest --env node --verbose --runInBand --bail",
-    "test-unit": "yarn test test/unit.*test.*",
-    "test-integration-once": "yarn test test/integration.test.ts"
+    "test-unit": "turbo build --filter=@vercel/build-utils && pnpm test test/unit.*test.*",
```

Comment:
Why was the `turbo build` part necessary?

Reply:
Since these tests now require the built module, we must ensure its being built every time tests are run. turbo here lets that build hit the cache if nothing has changed. Can change to just `pnpm build` but then no caching will happen

Reply:
<picture data-single-emoji=":hmm:" title=":hmm:"><img class="emoji" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..TihGaWiL5akSxMkn.G9XtJWcsxjBSPtnZ10vRkGVPxe5Gg0ax247H_YBCGqzgZ1xY0yM14FU_Sjvbjrla6rFAsF06sG132xudaMAx9UFKj_PNuBHiZEumyR8HkAxWnL5BaA.TAPEtljwBzRzRekYMtbjkQ" alt=":hmm:" width="20" height="auto" align="absmiddle"></picture>

Reply:
Yeah so this does work when `turbo` is used to invoke `test-unit`, but if you call `pnpm test-unit` it wont run

Reply:
Yeah but thats expected. You should run `pnpm build` and then `pnpm test-unit` if you're running from the current directory.